packages/compass-assistant/test/assistant.eval.ts

```diff
@@ -90,6 +121,7 @@ function makeEvalCases(): ConversationEvalCase[] {
       expected: {
         messages: [{ text: c.expected, sources: c.expectedSources || [] }],
       },
+      tags: c.tags ?? [],
```

Comment:
tags are useful for looking at the evals as you iterate on them. i recommend inclusion